scene/3d/collision_object_3d.cpp

```diff
@@ -466,6 +479,7 @@ void CollisionObject3D::_bind_methods() {
 	ADD_GROUP("Collision", "collision_");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "collision_layer", PROPERTY_HINT_LAYERS_3D_PHYSICS), "set_collision_layer", "get_collision_layer");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "collision_mask", PROPERTY_HINT_LAYERS_3D_PHYSICS), "set_collision_mask", "get_collision_mask");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "collision_priority"), "set_collision_priority", "get_collision_priority");
```

Comment:
At least zero or negative values are undesirable, but I don't think it needs to be hint_range, since any number of other values could be valid. Even a large ratio is needed to reduce bounce.

Reply:
What do you think about checking in the setter if the value is < 1 (and force it to 1)?  I am afraid that some people try to put a value of 0 (maybe we can also add the information in the description)

Reply:
When there are many priority1 objects, there is a possibility to set a value less than 1 when placing a weaker object. So at least, the minimum value should be 0.001.

Reply:
For now, only simple errors have been implemented. If the value is forced to be modified, the implementation will be complicated because the priority value exists in two separate places: the interface object and the actual object.